testsuite/parse/ast-parse/literal/ast.txt

```diff
@@ -0,0 +1,47 @@
++ module
+  + function
+    + funcname
+      - id (f)
```

Comment:
What is the difference between `+` and `-` in the ast format?

Reply:
peglib shows a `-` if the node is a token, and `+` otherwise